react/src/components/toggle/SprkToggle.js

```diff
@@ -83,41 +100,69 @@ class SprkToggle extends Component {
 
 SprkToggle.defaultProps = {
   toggleIconName: 'chevron-down-circle',
-  contentId: uniqueId('sprk_toggle_content_'),
 };
 
 SprkToggle.propTypes = {
   /**
    * 	The name of the icon other than the default to use for the toggle.
    */
   toggleIconName: PropTypes.string,
+  /**
+   * Deprecated - use `triggerText` instead.
+   * The title text for the toggle.
+   */
+  // TODO: Remove title in issue #1296
+  title: PropTypes.string,
   /**
    * The title text for the toggle.
    */
-  title: PropTypes.string.isRequired,
+  triggerText: PropTypes.string.isRequired,
```

Comment:
Looks like we made triggerText required here but the old title prop wasnt. I think this might force people to upgrade to triggerText. We want to make sure to allow the old one until we delete it next release. This needs to be double checked.